File: benchmarks/scenarios/cross-session-persistence/anma/src/domains/accounts/service.py
"""accounts public surface. Kept decoupled from billing (see prior decision)."""

def get_user(user_id: str) -> dict:
    return {"id": user_id, "name": "Ada"}

def account_summary(user_id: str, invoiced: int = 0) -> dict:
    # `invoiced` is injected by the caller to keep accounts decoupled from billing
    u = get_user(user_id)
    return {**u, "invoiced": invoiced}

File: benchmarks/scenarios/ts-payments-v2/replay/anma/src/domains/accounts/service.ts
// accounts public surface.
export function getUser(userId: string) {
  return { id: userId, name: "Ada" };
}

export function accountSummary(userId: string, invoiced = 0) {
  // `invoiced` is injected by the caller to keep accounts decoupled from billing
  const u = getUser(userId);
  return { ...u, invoiced };
}
